### PR TITLE
Package for Electron app, including Installer and keybase binaries, dmg

### PR DESCRIPTION
**Pull request by gabriel**

### CODE REVIEW

File: desktop/app/installer.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+export default (callback) => {
+  const app = require('electron').app;
```
Comment (cjb) [inline desktop/app/installer.js]: I think you should be able to use ES2015 `import` in this file

File: desktop/app/installer.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const nslog = require('nslog');
+  const exec = require('child_process').exec;
+  const path = require("path");
+  const fs = require('fs');
+
+  // const exePath = app.getPath("exe")
```
Comment (cjb) [inline desktop/app/installer.js]: Maybe remove the commented-out lines or parameterize on debug?  We have a `__DEV__` but might not be set this early.

File: desktop/app/installer.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  fs.access(installerExec, fs.X_OK , function (err) {
+    if (err) {
+      // Installer is not available
+      callback(null)
+      return
+    }
+
+    var cmd = [installerExec, "--service-path="+servicePath, "--run-mode=prod"].join(" ")
+    exec(cmd, function(err, stdout, stderr) {
```
Comment (cjb) [inline desktop/app/installer.js]: Cool, tested that this doesn't break anything on Linux, app still starts up.

File: desktop/app/installer.js (hunk 2 above)
Comment (gabriel) [inline desktop/app/installer.js]: yeah, nuked

File: desktop/app/installer.js (hunk 1 above)
Comment (gabriel) [inline desktop/app/installer.js]: Updated imports syntax

File: desktop/app/installer.js (hunk 3 above)
Comment (gabriel) [inline desktop/app/installer.js]: Yeah was gonna special case to OSX but though checking for the Installer presence was good enough

**Comment by cjb** [comment]
LGTM!  Also pinging @keybase/react-hackers in case someone wants to take a look.

**Pull request merged**